modules/apps/jaggery/mdm/src/mdm/modules/user.js

```diff
@@ -420,25 +437,12 @@ var userModule = function () {
         }
         try {
             utility.startTenantFlow(carbonUser);
-
+            log.error("here")
             var url = mdmProps["httpsURL"] + "/mdm-admin/users/users-by-username";
-            var xhr = new XMLHttpRequest();
-            xhr.open("GET", url);
-            xhr.send();
-
-            var response = {};
-            if (xhr.status == 200 && xhr.readyState == 4) {
-                var responsePayload = parse(xhr.responseText);
-                var userList = responsePayload["responseContent"];
-                // generate response
-                response["status"] = "success";
-                response["content"] = userList;
-                return response;
-            } else {
-                // generate response
-                response["status"] = "error";
-                return response;
-            }
+            log.error("hmmmm");
```

Comment:
This was a fault due to that all the committed changes has been reverted and all the local changes has been re based.Issue has been fixed as soon as encountered.